packages/babel-traverse/test/regression-11350-hub-undefined.js

```diff
@@ -0,0 +1,20 @@
+import { parse } from "@babel/parser";
+
+import _traverse from "../lib/index.js";
+const traverse = _traverse.default || _traverse;
+
+describe("this.hub should not be undefined while traversing a program or file", function () {
+  it("traverse", function () {
+    const p = parse("try { throw new Error() } catch (e) { const v = 0 }");
+    traverse(p, {
+      CatchClause(path) {
+        path.node.param.name = "v";
+        expect(function () {
+          path.scope.crawl();
+        }).toThrow(TypeError('Duplicate declaration "v"'));
```

Comment:
Here the test relies on the scope check and the specific message thrown during the scope check. Could you construct a minimal unit test instead? For example you can just call `path.hub.buildError` with a custom error message, which is also the argument of the unit test assertion.